usr/share/rear/restore/NFS_SERVER/default/400_restore_with_nfs_server.sh

```diff
@@ -0,0 +1,25 @@
+# 400_restore_with_nfs_server.sh
+
+FILE=$TARGET_FS_ROOT/rear_finished.txt
+
+NFS_PORT=2049
+
+Print "Mount the nfs share: 'mount -t nfs -o nfsvers=4 <ip>:/ <destination>' and restore all files to mounted destination."
+Print "Create the $FILE file when the restore is completed and umount the share."
+
+rm -rf $FILE
+
+Print "Wait until $FILE was created and there is no connection on the NFS-Port $NFS_PORT."
+
+#  or look at /var/lib/nfs/rmtab
+nfs_connections=$(ss -tanpH state established "( sport = $NFS_PORT )" | wc -l)
+
+while [ ! -f "$FILE" ] || [ "$nfs_connections" -gt 0 ]; do
+    sleep 5
+    nfs_connections=$(ss -tanpH state established "( sport = $NFS_PORT )" | wc -l)
+done
```

Comment:
For this `while` loop I'd suggest adding a disk free info for the mounted filesystems to serve as a progress indicator. That way one can watch the restore progress in terms of MB written to disk.

Reply:
Use `df --total --local -h` and extract the total value from the last line, this works on all supported distros.